modules/API/API.js

```diff
@@ -1046,8 +1046,20 @@ function initCommands() {
             callback(conference?.getMeetingUniqueId() || '');
             break;
         }
-        case '_new_electron_screensharing_supported': {
-            callback(true);
+        case 'open-desktop-picker': {
+            const { desktopSharingSources } = APP.store.getState()['features/base/config'];
```

Comment:
Does it make sense to use:
```
APP.store.dispatch(showDesktopPicker(options, onSourceChoose));
```

instead of the global one? This way one day we can remove the `window.JitsiMeetScreenObtainer` all together.

Reply:
Sure, I can do that!

Reply:
Done, PTAL!